scripts/pre-commit

```diff
@@ -1,2 +1,3 @@
 #!/bin/sh
 make lint
+make update-version
```

Comment:
Thinking about this again I don't think this is really required, instead as part of taking a release the update-version target can be manually executed to avoid updating the version each time a commit is made. This also avoids the issue where two commits could have the same version number on a feature branch, but have different version numbers when merged into develop (as more commits can be added to develop whilst the feature is in progress)